sig-security/security-audit-2021/RFP.md

```diff
@@ -0,0 +1,121 @@
+# Request for Proposal
+
+## Kubernetes Third-Party Security Audit
+
+The Kubernetes Third-Party Audit Working Group (working group, henceforth) is soliciting proposals from Information Security vendors for a comprehensive security audit of the Kubernetes Project.
+
+### Background
+
+In August of 2019, the Kubernetes Security Audit working group, in concert with the CNCF, Trail of Bits, and Atredis Partners, completed the first comprehensive security audit of the Kubernetes project’s [codebase](https://github.com/kubernetes/kubernetes/), working from version 1.13.
+
+These findings, below, paint a broad picture of Kubernetes security, as of version 1.13, and highlight some areas that warrant further, deeper, research.
+
+* [Kubernetes Security Review](../security-audit-2019/findings/Kubernetes%20Final%20Report.pdf)
+* [Attacking and Defending Kubernetes Installations](../security-audit-2019/findings/AtredisPartners_Attacking_Kubernetes-v1.0.pdf)
+* [Whitepaper](../security-audit-2019/findings/Kubernetes%20White%20Paper.pdf)
+* [Threat Model](../security-audit-2019/findings/Kubernetes%20Threat%20Model.pdf)
+
+### Project Goals and Scope
+
+This subsequent audit is intended to be the second in a series of recurring audits, each focusing on a specific aspect of Kubernetes while maintaining coverage of all aspects that have changed since the previous audit ([1.13](../security-audit-2019/findings/)).
+
+The scope of this audit is the most recent release (1.20) of the core [Kubernetes project](https://github.com/kubernetes/kubernetes).
```

Comment:
secrets-store-csi-driver is in https://github.com/kubernetes-sigs/secrets-store-csi-driver rather than k/k; You may want to amend this to say "and certain other code maintained by [Kubernetes SIGs](https://github.com/kubernetes-sigs/)" just to make sure they don't get excluded from the scope.

Reply:
done.